kotlinx-coroutines-core/jvm/src/module-info.java

```diff
@@ -0,0 +1,23 @@
+module kotlinx.coroutines.core {
+    requires transitive kotlin.stdlib;
+    requires kotlinx.atomicfu;
+
+    // these are used by kotlinx.coroutines.debug.AgentPremain
+    requires static java.instrument; // contains java.lang.instrument.*
+    requires static jdk.unsupported; // contains sun.misc.Signal
+
+    exports kotlinx.coroutines;
+    exports kotlinx.coroutines.channels;
+    exports kotlinx.coroutines.debug;
+    exports kotlinx.coroutines.debug.internal;
+    exports kotlinx.coroutines.flow;
+    exports kotlinx.coroutines.flow.internal;
```

Comment:
Could you please elaborate on how exactly you got these packages?

It seems like some of them should not be exported in the first place -- all `internal` (though I have to double check some of them), `scheduling` and `intrinsics`

Reply:
These packages are mostly used in `reactive/kotlinx-coroutines-reactive/src/ReactiveFlow.kt`. So I need to export them from core so I don't get a compile error in `kotlinx-coroutines-reactive`.